src/lightning_app/core/app.py

```diff
@@ -49,6 +49,7 @@ def __init__(
         root: "lightning_app.LightningFlow",
         debug: bool = False,
         info: frontend.AppInfo = None,
+        root_path: str = "",
```

Comment:
the text below says that for customization, I should set it to "/path". By extension, shouldn't this then default to "/"?

Reply:
uvicorn and fastapi both have empty string as default:
https://github.com/encode/uvicorn/blob/master/uvicorn/main.py#L248
https://github.com/tiangolo/fastapi/blob/master/fastapi/applications.py#L77